src/libs/Navigation/AppNavigator/createCustomStackNavigator/CustomRouter.js

```diff
@@ -12,11 +12,11 @@ const isAtLeastOneCentralPaneNavigatorInState = (state) => _.find(state.routes,
 
 /**
  * @param {Object} state - react-navigation state
- * @returns {String|undefined}
+ * @returns {String}
```

Comment:
```suggestion
 * @returns {String|undefined}
```
This can still return undefined when `!state` is true

Reply:
That returns an empty string with the new logic, no? Am I missing something?

```
    if (!state) {
        return '';
    }
```

Reply:
Oh sorry